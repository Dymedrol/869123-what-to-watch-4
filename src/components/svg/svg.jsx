import React from "react";

export const PlayS = () => {
  return <svg width="19" height="19" viewBox="0 0 19 19" fill="none">
    <path fillRule="evenodd" clipRule="evenodd" d="M0 0L19 9.5L0 19V0Z" fill="#EEE5B5"/>
  </svg>;
};

export const Add = () => {
  return <svg width="19px" height="20px" viewBox="0 0 19 20" version="1.1">
    <title>+</title>
    <desc>Created with Sketch.</desc>
    <g id="Page-1" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
      <polygon id="+" fill="#EEE5B5" points="10.777832 11.2880859 10.777832 19.5527344 8.41650391 19.5527344 8.41650391 11.2880859 0.627929688 11.2880859 0.627929688 8.92675781 8.41650391 8.92675781 8.41650391 0.662109375 10.777832 0.662109375 10.777832 8.92675781 18.5664062 8.92675781 18.5664062 11.2880859"></polygon>
    </g>
  </svg>;
};

