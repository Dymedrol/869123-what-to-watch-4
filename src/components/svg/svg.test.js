import React from 'react';
import renderer from 'react-test-renderer';
import {PlayS, Add} from './svg.jsx';

it(`Проверка снепшота компонента PlayS`, () => {
  const tree = renderer.create(
      <PlayS/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

it(`Проверка снепшота компонента Add`, () => {
  const tree = renderer.create(
      <Add/>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
