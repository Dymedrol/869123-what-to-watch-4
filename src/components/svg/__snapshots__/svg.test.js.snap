// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Проверка снепшота компонента Add 1`] = `
<svg
  height="20px"
  version="1.1"
  viewBox="0 0 19 20"
  width="19px"
>
  <title>
    +
  </title>
  <desc>
    Created with Sketch.
  </desc>
  <g
    fill="none"
    fillRule="evenodd"
    id="Page-1"
    stroke="none"
    strokeWidth="1"
  >
    <polygon
      fill="#EEE5B5"
      id="+"
      points="10.777832 11.2880859 10.777832 19.5527344 8.41650391 19.5527344 8.41650391 11.2880859 0.627929688 11.2880859 0.627929688 8.92675781 8.41650391 8.92675781 8.41650391 0.662109375 10.777832 0.662109375 10.777832 8.92675781 18.5664062 8.92675781 18.5664062 11.2880859"
    />
  </g>
</svg>
`;

exports[`Проверка снепшота компонента PlayS 1`] = `
<svg
  fill="none"
  height="19"
  viewBox="0 0 19 19"
  width="19"
>
  <path
    clipRule="evenodd"
    d="M0 0L19 9.5L0 19V0Z"
    fill="#EEE5B5"
    fillRule="evenodd"
  />
</svg>
`;
